runelite-client/src/main/java/net/runelite/client/plugins/itemstats/ItemStatChanges.java

```diff
@@ -352,6 +365,19 @@ private void init()
 		add(combo(heal(HITPOINTS, perc(.15, 1)), heal(RUN_ENERGY, 100)), BANDAGES_25202);
 		add(combo(boost(ATTACK, perc(.15, 5)), boost(STRENGTH, perc(.15, 5)), boost(DEFENCE, perc(.15, 5)), boost(RANGED, perc(.15, 5)), boost(MAGIC, perc(.15, 5)), heal(PRAYER, perc(.25, 8))), POTION_OF_POWER1, POTION_OF_POWER2, POTION_OF_POWER3, POTION_OF_POWER4);
 
+		// Moths & Butterflies
+		final Effect rubyHarvest = boost(ATTACK, perc(.15, 4));
+		final Effect snowyKnight = heal(HITPOINTS, perc(.15, 4));
+		final Effect blackWarlock = boost(STRENGTH, perc(.15, 4));
+		final Effect sapphireGlacial = boost(DEFENCE, perc(.15, 4));
+		final SunlightMoth sunlightMoth = new SunlightMoth(.2, 6);
+		add(rubyHarvest, RUBY_HARVEST, RUBY_HARVEST_MIX_1, RUBY_HARVEST_MIX_2);
+		add(snowyKnight, SNOWY_KNIGHT, SNOWY_KNIGHT_MIX_1, SNOWY_KNIGHT_MIX_2);
```

Comment:
Multi-bite/use food items (such as cakes and pies) should list the amount each use heals, not the total. This would incorrectly list a 1-dose Snowy knight mix as healing twice as much as it does.